VisualFSharp.sln

```diff
@@ -13,6 +13,9 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "VisualFSharp", "VisualFShar
 	EndProjectSection
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Compiler", "Compiler", "{3881429D-A97A-49EB-B7AE-A82BA5FE9C77}"
+	ProjectSection(SolutionItems) = preProject
+		src\Compiler\FSComp.txt = src\Compiler\FSComp.txt
+	EndProjectSection
```

Comment:
Should also probably add to the service project as a "None" item.

Reply:
And to FSharp.sln and service solutions too.

Reply:
Oh, good.

Reply:
It's already [somehow](https://github.com/dotnet/fsharp/blob/0d88a9cebc99b5685e2418d6b0981b39bd9d0750/src/Compiler/FSharp.Compiler.Service.fsproj#L96) added here, should we still change this?

Reply:
No, that's ok. Can you add it to other two solutions please?

Reply:
Sorry the thread wasn't getting updated for me so I wrote the same thing :)
Yep, added to both.